WEEK1-B/kwonjeong/boj12605.cpp

```diff
@@ -0,0 +1,28 @@
+// BOJ 12605번 단어순서 뒤집기
+#include <iostream>
```

Comment:
```suggestion
#include <bits/stdc++.h>
```
 헤더를 사용하면 더 좋아요 (속도 향상 및 심플)

Reply:
오왕 감사합니다!!

Reply:
`main` 함수 시작 직후

```cpp
ios_base::sync_with_stdio(false);
cin.tie(nullptr);
cout.tie(nullptr);
```
를 해주면 입출력을 더 빠르게 처리할 수 있어요

또한, 개행문자를 출력할 때, `iostream`의 `std::endl` 객체를 사용하는 대신, 개행 문자 리터럴 `"\n"`을 직접 출력하는 것이 더 실행 속도가 빠릅니다!

Reply:
아 endl 보다 "\n" 쓰는게 실행 시간 제한이 있을 때 더 좋겠네요! 감사합니다! 말씀해주신 코드도 앞으로 써 볼게요!